Packages/com.vrchat.ClientSim/Runtime/CameraStacking/ClientSimStackedVRCameraSystem.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.XR;
+using VRC.Core;
+using VRC.SDK3.ClientSim;
+
+namespace VRC.SDK3.ClientSim
+{
+    public class ClientSimStackedVRCameraSystem : MonoBehaviour
+    {
+        public ClientSimStackedCamera[] CameraStack;
+        [HideInInspector] public bool IsCameraStackingEnabled = false;
+        
+        private Camera MainSceneCamera;
+        private bool IsInitialized = false;
+        private bool IsReady = false;
+        private List<Camera> Cameras;
+        
+        private ClientSimMenu clientSimMenu;
+
+        public void Initialize(Camera playerCamera, ClientSimMenu menu)
+        {
+            MainSceneCamera = playerCamera;
+            clientSimMenu = menu;
+        }
+
+        public void Ready()
+        {
+            IsReady = true;
+        }
+        
+        void Update()
+        {
+            if(!IsReady) return;
+            if (!IsInitialized) { InitializeStackedSystem(); }
+            else
+            {
+                if (!Cameras[0].enabled)
+                {
+                    Debug.LogError("Stacked Cameras are not enabled.");
```

Comment:
Could we add something to this error message to help users? They probably don't know what this means or why it's an error.

Reply:
I'll remove it entirely, I think it was only needed to handle android ifdefs which have been removed